src/test/java/core/basesyntax/strategy/impl/BalanceOperationTest.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationsStrategy;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class BalanceOperationTest {
+    private static final String FRUIT_NAME = "banana";
+    private static final int FRUIT_QUANTITY = 120;
+    private static final int NEGATIVE_FRUIT_QUANTITY = -10;
```

Comment:
Don't make it like a constant, better use like a local variables directly in your tests